CMakeConfig.txt

```diff
@@ -80,6 +80,7 @@ set(PROTOCOL_SUNRISESET ON CACHE BOOL "support for the Sunrise / Sunset calculat
 set(PROTOCOL_TECHLICO ON CACHE BOOL "support for TechLiCo Lamps protocol")
 set(PROTOCOL_TEKNIHALL ON CACHE BOOL "support for the Teknihall weather station protocol")
 set(PROTOCOL_TFA ON CACHE BOOL "support for the TFA weather station protocol")
+set(PROTOCOL_UNITEC ON CACHE BOOL "support for the unitec switch protocol")
```

Comment:
Shouldn't PROTOCOL_UNITEC be PROTOCOL_UNITEC_SWITCH?
Since I understand that *Unitec* is the brand name and *switch* is the type of device. I believe there should be a difference between device types, since apparently there is also a [push button switch] (https://www.amazon.co.uk/Unitec-47195-Button-Switch-Radio/dp/B00E625IE6?ie=UTF8&*Version*=1&*entries*=0)